src/VisualStudio/TestUtilities2/VisualStudioTestCompositions.vb

```diff
@@ -19,7 +21,8 @@ Namespace Microsoft.VisualStudio.LanguageServices.UnitTests
             AddAssemblies(
                 GetType(ServicesVSResources).Assembly,
                 GetType(CSharpVSResources).Assembly,
-                GetType(BasicVSResources).Assembly).
+                GetType(BasicVSResources).Assembly,
+                GetType(EditorConfigGenerator).Assembly).
```

Comment:
>EditorConfigGenerator [](http://example.com/codeflow?start=24&length=21)

Generally we use the Resources type when adding assembly to composition. This is because types may easily move around to different assemblies changing the composition, where as the resource types are very specific to each assembly.

Reply:
I don't see much value in testing the two lines in the EditorConfigGenerator:
``` 
var groupOptions = _editorConfigGenerator.GetDefaultOptions(language);
return EditorConfigFileGenerator.Generate(groupOptions, _globalOptions, language);
```

The tests use EditorConfigFileGenerator API directly without the external access layer being involved.